.github/workflows/build-bootc.yaml

```diff
@@ -194,3 +204,21 @@ jobs:
             -e "BUILDAH_URL=$URL:bootc" \
             quay.io/buildah/stable:v1.36.0 \
               /bootc-images/buildah-build-and-push-manifest.sh
+          
+          DIGEST=$(cat bootc-images/digest)
+          echo "digest=$DIGEST" >> $GITHUB_OUTPUT
+
+      - name: Install cosign
+        uses: sigstore/cosign-installer@v3.5.0
```

Comment:
Would signing with Podman be equivalent (see [docs](https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/8/html/building_running_and_managing_containers/assembly_signing-container-images_building-running-and-managing-containers#signing-container-images-with-sigstore-signatures-using-fulcio-and-rekor_assembly_signing-container-images)) or where is the diff, DYK?

Reply:
It looks like using Podman would require considerably more configuration. cosign will use sigstore's Fulcio and Rektor servers by default and will also automatically determine the identity of the signer. All of these things would need to be explicitly provided in /etc to Podman.